src/common/transformations/tests/op_conversions/convert_interpolate11_downgrade_test.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <openvino/opsets/opset11.hpp>
+#include <openvino/opsets/opset4.hpp>
+#include <openvino/pass/manager.hpp>
+#include <transformations/op_conversions/convert_interpolate11_downgrade.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+
+TEST_F(TransformationTestsF, ConvertInterpolate11ToInterpolate4_scales) {
```

Comment:
it seems that the tests are almost the same, is it possible to parameterize them?
or  move model_creation and ref_model_creation to the separate functions and just pass attributes to these functions?

Reply:
Refactored with model building functions